### PR TITLE
chore: update price-chart api for production

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

**Review by nguyenhoaidanh — APPROVED** [review]
lgtm

**Review by nguyenhuudungz — APPROVED** [review]

File: src/hooks/useBasicChartData.ts [hunk 1]
```diff
@@ -189,7 +183,7 @@ export default function useBasicChartData(tokens: (Token | null | undefined)[],
     !isKyberDataNotValid && kyberData && chainId
       ? liveDataApi[chainId] + `?ids=${tokenAddresses[0]},${tokenAddresses[1]}`
       : null,
-    fetchKyberDataSWRWithHeader,
+    fetchKyberDataSWR,
```
Comment (viet-nv) [inline src/hooks/useBasicChartData.ts:186]: for aggregator api, we must add accept-version header

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/hooks/useBasicChartData.ts (hunk 1 above)
Comment (XiaoYhun) [reply src/hooks/useBasicChartData.ts:186]: I just merged both functions into one, no need to have multi versions of this

Comment (viet-nv) [reply src/hooks/useBasicChartData.ts:186]: no, if domain is price-chart, accept-version will break the api call

Comment (viet-nv) [reply src/hooks/useBasicChartData.ts:186]: image

Comment (viet-nv) [reply src/hooks/useBasicChartData.ts:186]: only aggregator api accept header `accept-version`

Comment (XiaoYhun) [reply src/hooks/useBasicChartData.ts:186]: ah only for aggregator api, my bad

**Review by nguyenhuudungz — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**